src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/PackageWrapper.cs

```diff
@@ -32,22 +33,26 @@ public PackageWrapper(string Name, string FullName, string FamilyName, bool IsFr
 
         public static PackageWrapper GetWrapperFromPackage(Package package)
         {
+            string path;
             try
             {
-                return new PackageWrapper(
-                        package.Id.Name,
-                        package.Id.FullName,
-                        package.Id.FamilyName,
-                        package.IsFramework, 
-                        package.IsDevelopmentMode,
-                        package.InstalledLocation.Path
-                        );
+                 path = package.InstalledLocation.Path;
```

Comment:
I feel that we can do a simple null check here rather than handling the exception. This doesn't look like an exception as we know that `Path` might not be set.

Reply:
It is not the case that some value is null. `package` is not null. However, package.InstalledLocation throws this exception-
![image](https://user-images.githubusercontent.com/28739210/87998301-5fe12e00-caac-11ea-9c65-01983c9521fe.png)

As mentioned in the description of this PR, there are different classes derived from the `ArgumentExceptions` class, two of them being `ArgumentNullException` and `ArgumentOutOfRange` exception. It is advised that we try to catch one of these. I did try to do a null check as well as to try and catch the above two exceptions but the exception being thrown wasn't of that type. Hence, I had to use `ArgumentException`. 
Ref- https://docs.microsoft.com/en-us/dotnet/api/system.argumentexception?view=netcore-3.1

Reply:
Got it! Thanks for clarification.